presto-main/src/main/java/com/facebook/presto/operator/aggregation/PrecisionRecallAggregation.java

```diff
@@ -101,8 +102,8 @@ public static void combine(
             @AggregationState PrecisionRecallState otherState)
     {
         if (state.getTrueWeights() == null && otherState.getTrueWeights() != null) {
-            state.setTrueWeights(new FixedDoubleHistogram(otherState.getTrueWeights().clone()));
-            state.setFalseWeights(new FixedDoubleHistogram(otherState.getFalseWeights().clone()));
+            state.setTrueWeights(otherState.getTrueWeights().clone());
```

Comment:
nit: why we are using `.clone()` instead of `new FixedDoubleHistogram(otherState.getTrueWeights))` ?